### DISCUSSION TITLE
CommunityToolkit8.1安装后报错（基于VS2017、.Net Standard V2.0）

### DISCUSSION THREAD

**Discussion by mingjun0315** [BYJRK/DotNet-Discussions#16] [Q&A]
大佬大佬，安装CommunityToolkit8.1后报错：静态调用语法"[MSBuild]:VersionLessThan($(MVVMToolkitCurrentCompilerVersion),4.O)"无效.未找到方法"[MSBuild]::VersionLessThan"。只能回退到7.1.2版本才能消除报错，但这样的话就缺失很多功能了。请问是我VS的版本问题吗？还是我有其他操作没有弄好。

**Comment by BYJRK** [comment]
你的VS是什么版本？这个Toolkit对于Roslyn的版本之类也是有要求的

**Reply by mingjun0315** [reply]
VS2017专业版

**Reply by BYJRK** [reply]
VS2017的Roslyn版本太久了，你恐怕需要考虑用最新版
